src/Send/SendScreen/utils.ts

```diff
@@ -175,14 +176,11 @@ export const recomputeAll = async ({
           defaultAsset,
           selectedTokenInfo,
         )
-        _fee = new MultiToken(yoroiUnsignedTx.unsignedTx.fee.values, {
-          defaultNetworkId: yoroiUnsignedTx.unsignedTx.fee.defaults.networkId,
-          defaultIdentifier: yoroiUnsignedTx.unsignedTx.fee.defaults.identifier,
-        })
-        balanceAfter = tokenBalance.getDefault().minus(parsedAmount).minus(minAda).minus(_fee.getDefault())
+
+        fee = yoroiUnsignedTx.fee[defaultAsset.identifier]
```

Comment:
is there any chance for the `fee` to be `undefined` ? cuz we don't have tests that covered `Quantities` operations with `undefined`, since it only accepts `Quantity` as parameters, but if it happens to initialize a `BigNumber` with `undefined` will prevent all further operations by always returning `NaN`.